modules/nw-metallb-operator-setting-pod-priority-affinity.adoc

```diff
@@ -5,26 +5,28 @@
 [id="nw-metallb-operator-setting-pod-priority-affinity_{context}"]
 = Configuring pod priority and pod affinity in a MetalLB deployment
 
-You can optionally assign pod priority and pod affinity rules to `controller` and `speaker` pods by configuring the `MetalLB` custom resource. The pod priority indicates the relative importance of a pod on a node and schedules the pod based on this priority. Set a high priority on your `controller` or `speaker` pod to ensure scheduling priority over other pods on the node. 
+You can optionally assign pod priority and pod affinity rules to `controller` and `speaker` pods by configuring the `MetalLB` custom resource. The pod priority indicates the relative importance of a pod on a node and schedules the pod based on this priority. Set a high priority on your `controller` or `speaker` pod to ensure scheduling priority over other pods on the node.
 
-Pod affinity manages relationships among pods. Assign pod affinity to the `controller` or `speaker` pods to control on what node the scheduler places the pod in the context of pod relationships. For example, you can allow pods with logically related workloads on the same node, or ensure pods with conflicting workloads are on separate nodes. 
+Pod affinity manages relationships among pods. Assign pod affinity to the `controller` or `speaker` pods to control on what node the scheduler places the pod in the context of pod relationships. For example, using pod affinity rules, you can ensure that these pods are co-located on the same nodes or nodes. This can help optimize network communication and reduce latency between these components.
 
 .Prerequisites
 
 * You are logged in as a user with `cluster-admin` privileges.
 
 * You have installed the MetalLB Operator.
 
+* You have started MetalLB on your cluster.
+
 .Procedure
-. Create a `PriorityClass` custom resource, such as `myPriorityClass.yaml`, to configure the priority level. This example uses a high-priority class:
+. Create a `PriorityClass` custom resource, such as `myPriorityClass.yaml`, to configure the priority level. This example defines a `PriorityClass` named `high-priority` with a value of `1000000`. Pods that are assigned this priority class will be considered higher-priority during scheduling compared to pods with lower priority classes:
```

Comment:
will be > are

![Screenshot from 2023-08-31 12-09-18](https://github.com/openshift/openshift-docs/assets/57954076/7398b463-5ad6-43c4-9cb0-9f727e18b3b5)